### PR TITLE
Added CustomerService

### PR DESCRIPTION
**Pull request by isfandiyoro5**
Updated apsetting.json

### CODE REVIEW

File: OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/CustomerService.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            _customerRepository = customerRepository;
+            _mapper = mapper;
+        }
+
+        public async Task<int> AddCustomerAsync(CustomerRequestDTO customerRequestDTO)
+        {
+            return await _customerRepository.AddCustomerAsync(_mapper.Map<Customer>(customerRequestDTO));
+        }
+
+        public async Task<int> DeleteCustomerAsync(CustomerRequestDTO customerRequestDTO, int id)
```
Comment (MirazamTUT) [inline OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/CustomerService.cs]: id ni o'zi qolishi kerak

File: OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/CustomerService.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        {
+            var customerResult = await _customerRepository.GetCustomerByIdAsync(id);
+            if(customerResult is not null) 
+            {
+                customerResult = _mapper.Map<Customer>(customerRequestDTO);
```
Comment (MirazamTUT) [inline OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/CustomerService.cs]: shunda bula keremas

File: OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/CustomerService.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                customerResult.CustomerId = id;
```
Comment (MirazamTUT) [inline OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/CustomerService.cs]: delete methodida id bo'yicha qidirib bor bo'lsa o'chiramz

**Review by MirazamTUT — CHANGES_REQUESTED** [review]

File: OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/CustomerService.cs (hunk 2 above)
Comment (isfandiyoro5) [reply OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/CustomerService.cs]: ok

File: OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/CustomerService.cs (hunk 3 above)
Comment (isfandiyoro5) [reply OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/CustomerService.cs]: ok

File: OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/ServiceExtentions/ServiceExtentions.cs [hunk 4]
```diff
@@ -18,6 +18,7 @@ public static void RegisterServices(this IServiceCollection services, IConfigura
             services.AddScoped<IReportRepository, ReportRepository>();
             services.AddScoped<ICustomerRepository, CustomerRepository>();
             services.AddScoped<IPurchaseRepository, PurchaseRepository>();
```
Comment (MirazamTUT) [inline OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/ServiceExtentions/ServiceExtentions.cs]: mashetga PharmacyRepositoryni ham registratsiya qivoring yozilmagan ekan

**Review by MirazamTUT — CHANGES_REQUESTED** [review]

**Review by MirazamTUT — DISMISSED** [review]

**Review by MirazamTUT — APPROVED** [review]

**Pull request merged**